packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -465,6 +450,8 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
         >
           {errorMessage !== undefined ? errorMessage : ''}
         </div>
+        <EventListener targetRef={this._comboBoxWrapper} type="focus" listener={this._onResolveOptions} capture />
```

Comment:
Will this work since the ref won't be set on first render?

Reply:
I believe so because the hook checks for undefined: https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/fluentui/react-component-event-listener/src/useEventListener.ts#L38

Reply:
But if the ref is undefined, the event listener won't actually be added until the next time the component is rendered. So even though it won't throw, it won't do what is expected.

Reply:
Actually never mind, I was wrong: the actual adding of the event listener is done in useEffect, Which happens after rendering. So this is fine.

Reply:
I think this is a problem actually because the test is currently failing: https://uifabric.visualstudio.com/fabricpublic/_build/results?buildId=53154&view=logs&j=92058b04-f16a-5035-546c-cae3ad5e2f5f&t=17e772b0-f0c3-53ab-7aac-ad6df80e30d7

> office-ui-fabric-react:       Error: Uncaught [Error: @fluentui/react-component-event-listener: Passed `element` is not valid or does not support `addEventListener()` method.]

So I think `this._comboBoxWrapper` is the `element` that doesn't support the `addEventListener()` method call inside useEventListener.ts @layershifter thoughts on how to work around this?

Reply:
```tsx
<KeytipData keytipProps={keytipProps} disabled={disabled}>
  {keytipAttributes => (
    <>
      <div ref={this._comboBoxWrapper} /> // Order there is important see
      <EventListener
        targetRef={this._comboBoxWrapper}
        type="focus"
        listener={this._onResolveOptions}
        capture
      />
      <EventListener
        targetRef={this._comboBoxWrapper}
        type="pointerdown"
        listener={this._onPointerDown}
        capture
      />
    </>
  )}
</KeytipData>
```

See https://github.com/OfficeDev/office-ui-fabric-react/pull/11949/files#r382090349 for more details.